### PR TITLE
Verified it works, added some clarity re: workspaces

### PR DESCRIPTION
**Pull request by l0010o0001l**
Also bumped Config section up since that's the thing that happens as soon as you hit `time_log_robot`. Removed `\ -X GET` from `curl` because those attributes are already implied via `curl`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,25 @@ The simplest usage is just to invoke the robot:
 
     $ time_log_robot
 
+### Configuration
```
Comment (supremebeing7) [inline README.md]: Good idea on the rearrange. I'd suggest that the chunk of text lines 102-115 ("On start, the robot will ask you for these details..." through "...so open up that file and edit to your heart's content") should go here.

**Comment by l0010o0001l** [comment]
Hope is a dangerous thing my friend, it can kill a man... - _Ellis Boy "Red" Redding_

**Comment by supremebeing7** [comment]
Bangarang

Thanks @l0010o0001l !

**Pull request merged**